src/main/java/org/folio/des/service/JobExecutionService.java

```diff
@@ -76,9 +110,14 @@ protected void validateIncomingExportConfig(Job job) {
     });
   }
 
-  private JobCommand buildBaseJobCommand(Job job) {
+  private JobCommand buildBaseJobCommand(Job job, JobCommand.Type type) {
     var result = new JobCommand();
-    result.setType(JobCommand.Type.START);
+    if(type == JobCommand.Type.START) {
+      result.setType(type);
+    }
+    else if(type == JobCommand.Type.RESEND) {
+      result.setType(type);
+    }
```

Comment:
Here is simpler version: result.setType(type);

Reply:
fixed